sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/MethodDeclarationTracker.cs

```diff
@@ -35,7 +36,7 @@ protected MethodDeclarationTracker(IAnalyzerConfiguration analysisConfiguration,
         {
         }
 
-        protected abstract SyntaxToken GetMethodIdentifier(SyntaxNode methodDeclaration);
+        protected abstract SyntaxToken? GetMethodIdentifier(SyntaxNode methodDeclaration);
```

Comment:
MethodIdentifier is null when the method is not a "normal" method, e.g. a property for example